Services/DataX.Flow/DataX.Flow.SchemaInference/Kafka/KafkaMessageBus.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace DataX.Flow.SchemaInference.Kafka
 {
-    public class KafkaMessageBus : IMessageBus
+    public class KafkaMessageBus : IMessageBus, IDisposable
```

Comment:
I assume these are good things to be implementing, were these reported anywhere?

Reply:
The roslyn analyzer detects this and displays a message in the output/ error list panel.